### PR TITLE
Create Readme_ES.md

### PR DESCRIPTION
**Pull request by LuigiBaute**

### CODE REVIEW

**Comment by bartimaeusnek** [comment]
Are the Translated names of the mods really correct? from 110 onwards? I.e. bayas ++, since its a name, it shouldnt be translated imo...

**Comment by Dream-Master** [comment]
Yes Mods not need to translated.  Rest is fine I guess.

File: Readme_ES.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
+# Paquete GT para moderadores de New Horizons
```
Comment (Traxis) [inline Readme_ES.md]: Let's do some literal translations here: "GT Package for moderators of New Horizons".

File: Readme_ES.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+## Paquete para moderadores verificado a través de un ajuste a los moderadores (LOS MODERADORES OBTIENEN AUTORIZACIÓN)
```
Comment (Traxis) [inline Readme_ES.md]: Another one: "Package for moderators verified through a setting to the moderators (THE MODERATORS OBTAIN AUTHORIZATION)".

File: Readme_ES.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+
+
+Versión 2.0.2.7 is out 15.01.2017
```
Comment (Traxis) [inline Readme_ES.md]: At least translate "is out"...

File: Readme_ES.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+
+[![estado de compilación](http://jenkins.usrv.de:8081/buildStatus/icon?job=GTNewHorizons%20Configs)](http://jenkins.usrv.de:8081/job/GTNewHorizons%20Configs/)
+
+## Un paquete para moderadores más grande con cada especificación verificada
```
Comment (Traxis) [inline Readme_ES.md]: But there's more: "A package for moderators bigger with every verified specification".

**Comment by LuigiBaute** [comment]
I can made all the changes till you guys are pleased. and explain me the context.

**Pull request merged**